src/services/dashboards/widgets/aws-cloud-front-cost/AWSCloudFrontCost.vue

```diff
@@ -96,7 +96,8 @@ const fetchData = async () => new Promise((resolve) => {
 
 const drawChart = (chartData) => {
     const { chart, xAxis, yAxis } = createXYVerticalChart();
-    setChartColors(chart, state.colorSet);
+    // TODO: Temporary annotation processing for refresh implementation
+    // setChartColors(chart, state.colorSet);
```

Comment:
When using the AWS CloudFrontCost widget, colorSet causes a call stack overflow.
First of all, I processed the annotation to implement the Dashboard Refresh. 🙏 🙏 

![image](https://user-images.githubusercontent.com/50662170/208660707-d5e15c1c-333b-4f39-a20a-8d3c87652b54.png)

Reply:
Oops...I'll check it

Reply:
🙏 🙏 🙏